### ISSUE TITLE
Q3 CAMPD 508 scan remediation 

### ISSUE THREAD

**Issue by alangmaid** [US-EPA-CAMD/PI-1#6275]
508 testing results.  Need to triage and create development tickets for remediation.

Page / Screen
	
Description of Issue
	
Test(s)
	
508 Standard(s)
	
WCAG 2.0 Standard(s)

Data > Query Data > Filters
	
Vintage Year (Under Allowance, Transactions) has no accessible name.  Returns error "Improper use of [aria-labelledby]: Referenced id not found.
	
5B
	
1194.21(l), 1194.22(n)
	
2.4.7, 2.4.6

Data > Query Data > Preview Data
	
The tagging on the sort buttons on the data table are reversed from their function.  Rather than reading what the button will do, it reads the current sort state.  When a column is already in ascending order, the button reads "sort by ascending." Clicking the button changes the sort to descending order.
 
See Figure 1
	
5B, 20A
	
1194.21(l), 1194.22(n)
	
2.4.7, 2.4.6, 4.1.1

FAQ
	
Link text should describe the purpose of the link. Include enough text to be descriptive.  Expand the following link text:
Under “Where can I learn more information about EPA’s regulatory programs?” Programs; Reporting
Under “Can EPA provide allowance price data or underlying datasets?” Progress
Under “Where can I learn more about EPA’s Allowance Trading Markets?” - Webpage
	
6A
	
1194.21(d), 1194.31(a), 1194.31(b)
	
2.4.4

Tutorials > Custom Data Download
 
Tutorials > Walkthrough 2-8
	
Document Title tag has not been set
	
12A
	
1194.31(a)
	
2.4.2

Tutorials > Main
	
There are two links marked Webinar Transcript Download - include the date in the link text for disambiguation.
	
6A
	
1194.21(d), 1194.31(a), 1194.31(b)
	
2.4.4

Tutorials > Walkthrough 8-11 > All
	
White text on blue-green background has insufficient contrast 3.1:1.
	
13C
	
1194.31(b)
	
1.4.3

Tutorials > Walkthrough 8-11 > Page 2
	
Points in right column of slide have no text layer and cannot be read.
	
7E
	
1194.21(f), 1194.22(a), 1194.31(a), 1194.31(b)
	
1.4.5

Tutorials > Walkthrough 8-11 > Page 2
	
Points in right column of slide have no text layer and cannot be read.
	
7E
	
1194.21(f), 1194.22(a), 1194.31(a), 1194.31(b)
	
1.4.5

Tutorials > Walkthrough 8-11 > Page 2
	
Each of the info boxes have white text on a colored background box.  These have insufficient contrast ranging from 2.4:1 to 2.96:1.
	
13C
	
1194.31(b)
	
1.4.3

Tutorials > Walkthrough 8-11 > Page 4
	
Each of the info boxes have white text on a colored background box.  These have insufficient contrast ranging from 2.1:1 to 2.8:1.
	
13C
	
1194.31(b)
	
1.4.3

Tutorials > Walkthrough 8-11 > Page 5
	
Link text on tan background has insufficient contrast 3.9:1.
	
13C
	
1194.31(b)
	
1.4.3

Tutorials > Walkthrough 8-11 > Page 6
	
Link text on blue-green background has insufficient contrast 2:1.
	
13C
	
1194.31(b)
	
1.4.3

Tutorials > Walkthrough 8-11 > Page 7
	
Link text on tan background has insufficient contrast 3.9:1.
	
13C
	
1194.31(b)
	
1.4.3

Viz Gallery
	
The tagging on the carousel buttons needs to be more descriptive i.e. "Carousel 1" or "Carousel 2" rather than just “1” or “2”.
	
5C
	
1194.21(l), 1194.22(n)
	
1.3.1

Viz Gallery
	
The link text "data" in paragraph one shares identical text to the data link in the top navigation.  These serve different purposes and must be clearly differentiated.
	
6A
	
1194.21(d), 1194.31(a), 1194.31(b)
	
2.4.4

Viz Gallery
	
The H2 headers in the carousel are identical to others on the page but does not convey how the two sections differ from each other.
 
See Note
	
10A
	
1194.21(d), 1194.31(a), 1194.31(b)
	
2.4.6

Viz Gallery
	
The text "1 of 2", "2 of 2" over the gallery image has insufficient contrast (ratio varies ~ 1.5:1)
 
See Figure 2
	
13C
	
1194.31(b)
	
1.4.3

[CAMPD Report.docx](https://github.com/user-attachments/files/16472169/CAMPD.Report.docx)

**Comment by mark-hayward-erg** [comment]
#6361 and #6362 have been added to address the issues with the data query and viz gallery.

EPA will address the issues found in the tutorials.

Closing this ticket.
